client/src/components/CardsContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import VideoCard from "./VideoCard";
+
+function CardsContainer({ videoData, setVideoData, setFetchData }) {
+  async function handleDelete(id) {
+    fetch(`http://localhost:5000/videos/${id}`, {
+      method: "delete",
+    })
+      .then((response) => response.json())
+      .then((result) => console.log(result))
+      .catch((error) => console.log("error", error));
+    setFetchData(true);
```

Comment:
What is `setFetchData` for? It's not super clear from its name what it's for, or what "true" vs "false" means for its value?

Reply:
I have removed setFetchData and added a new useState variable called "setVideoAdded" and set it as false and true when adding new videos data.